source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,103 @@
-## Updating registers
+## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#get-register)) and comparing the most recent entry number with your own copy.
+You can use the [`GET /register/` endpoint](#getregister) to get the current highest entry number in a register from the `total-entries` property. You can then regularly compare it with new queries to the register to see if there are any updates.
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+### Return all entries since your last update
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#get-download-register).
+You can use the [`GET /entries/` endpoint](#getentries) with the `start` and `limit` parameters to return specific entries to a register. 
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-records-key-entries):
+For example, the last highest entry number you got from the `country` register could be 205. You could then use the following request to find out if there are any new entries:
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/KIN/entries`
+Example request:
 
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/KIN/entries --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
+```http
+GET /entries?start=206/ HTTP/1.1
```

Comment:
Note that `limit` defaults to 100. So if there have been more than 100 entries added to the register they will not get all of them by calling `/entries?start206`. Is it worth adding something to specify this?

The user can look at the `Link` header returned. If there are more entries than the 100 returned then they would see a `Link` header with `rel="next"` returned:

```
<?start=306&limit=100>; rel="next"
```

And keep going until they no longer see a `Link` header with `rel="next"`.

Alternatively the user could use the `/register` endpoint to identify the new latest entry-number and calculate the correct limit. E.g. if the new `total-entries` value was `311` they should use `limit=105` (311-206). This will not work if the limit ends up being more than 5000 however, and they would need to follow the link headers anyway.

cc @arnau